atlasdb-config/src/main/java/com/palantir/atlasdb/http/errors/AtlasDbRemoteException.java

```diff
@@ -45,6 +51,19 @@ public String getErrorName() {
         return remoteException.getRemoteException().getErrorName();
     }
 
+    // Cannot be named getStackTrace() because that returns a list of (non-serializable) StackTraceElements.
```

Comment:
Why does that mean it cant have the same name? This returns the stack trace right and is not really overriding it. We have extra serializability here.